include/internal/cryptlib.h

```diff
@@ -96,6 +96,11 @@ void *openssl_fopen(const char *filename, const char *mode);
 
 uint32_t OPENSSL_rdtsc(void);
 
+int bn_prime_checks_for_size_ex(int prime_size, int security_bits);
+int bn_generate_prime_ex2(BIGNUM *ret, int bits, int safe,
+                          const BIGNUM *add, const BIGNUM *rem, BN_GENCB *cb,
+                          int checks);
```

Comment:
This is the wrong header file for these functions. Put them in crypto/include/internal/bn_int.h